bat-utils/lib/hapi-controllers-login.js

```diff
@@ -109,6 +100,27 @@ v1.logout = {
 }
 
 module.exports.routes = [
-  braveHapi.routes.async().path('/v1/login').whitelist().config(v1.login),
+  braveHapi.routes.async().method(['GET', 'POST']).path('/v1/login').whitelist().config(v1.login),
   braveHapi.routes.async().path('/v1/logout').config(v1.logout)
 ]
+
+module.exports.checkTeams = checkTeams
+
+async function checkTeams (runtime, credentials) {
+  const { organization } = runtime.login.github
+  const octokit = new Octokit({
+    debug: false,
+    auth: `token ${credentials.token}`
+  })
+
+  const { data: teams } = await octokit.teams.listForAuthenticatedUser({
+    org: organization
+  })
+
+  if (!teams.find(({ organization: { login } }) => login === organization)) {
+    throw boom.forbidden()
+  }
+  return {
+    scope: [organization]
```

Comment:
the scope here is different than it was before - previously the scope was one or more team names, not the organization :/

how can this be working?

Reply:
scope is not currently used. i just returned it in case we wanted to write a test or something else